src/js/src/main.js

```diff
@@ -2,13 +2,25 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import store from './store'
+import Axios from 'axios'
+import configs from './configs'
+
+import './assets/styles/main.scss'
 
 Vue.config.productionTip = false
 
-import './assets/styles/main.scss'
+Axios.get('services/frontend/properties/solrwaybackweb/')
+    .then(response => {
+        configs.playbackConfig.openwaybackBaseURL = response.data['openwayback.baseurl']
+        configs.playbackConfig.solrwaybackBaseURL = response.data['wayback.baseurl']
+        initializeVue()
+    })
+    .catch(error => initializeVue())
```

Comment:
Vi har ingen fallback for hvis vi ikke kan få fat i configs? (nu aner jeg ikke, er de altid tilgængelige?)

Reply:
Good objection! Nope - no fallback. The App will start though :) I think a smart way to do this would be to make a second request later on if the first one fails. I would like to implement this when I figure out a smart way to get the configs into a real store.

Reply:
True - all the more confusing when it starts and won't work properly from a user perspective tho. But i agree, it can wait with implementation. Also, this spawns a second task. Errors - we need some sort of error handling at least. The app might start here, but it should say something like "But we cant get the paths to wayback, seaching will work, but no display of individual pages" or something of that kind. Maybe. But we need error-handling, that's my point!

Reply:
I agree